src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/2.sql

```diff
@@ -1193,20 +1193,44 @@ AS
 
     BEGIN TRANSACTION
         
-        -- Check if tag exsit
         DECLARE @tagStatus TINYINT
         DECLARE @tagKey BIGINT
+
+        -- Lock the tag from external updates       
         SELECT @tagKey = TagKey, @tagStatus = TagStatus
         FROM dbo.CustomTag WITH(HOLDLOCK) 
         WHERE dbo.CustomTag.TagPath = @tagPath
 
+        -- Check existence
         IF @@ROWCOUNT = 0
             THROW 50404, 'custom tag not found', 1 
```

Comment:
can you confirm this throw dismisses the entire stored procedure and not just this transaction

Reply:
Hi Smitha, I tried add log in the 2nd transaction,  when exception is thrown, no output from 2nd.

I also tried with a test storage procedure
`CREATE PROCEDURE [dbo].[TestSP]
AS
BEGIN
	-- SET NOCOUNT ON added to prevent extra result sets from
	-- interfering with SELECT statements.
 SET NOCOUNT     ON
 SET XACT_ABORT  ON
Begin tran
print 's1';

THROW 50505, 'xxx',1;

print 'e1'
commit tran

Begin tran
print 's2'
print 'e2'
commit tran

END
`

I didn't see output from second tran.